app/javascript/components/miq-custom-tab/index.jsx

```diff
@@ -91,6 +107,23 @@ const MiqCustomTab = ({ containerId, tabLabels, type }) => {
     <Tab key={`tab${label}`} label={`${tabText(selectedLabels, label)}`} onClick={() => onTabSelect(label)} />
   ));
 
+  useEffect(() => {
+    if (activeTab) {
+      let elements = document.getElementsByClassName('bx--tabs--scrollable__nav-item');
+      elements.forEach((element) => {
+        element.classList.remove(activeTabClassName);
+        element.classList.remove(selectedClassName);
+      });
+      if (elements.length > 6) {
```

Comment:
Similarly here I don't like hardcoding the 6 - would be better if it could be dervived from an array like the settings const's size, for example.